src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float RESIZE_THRESHOLD = 0.75f;
+    private static final float RESIZE_MULTIPLIER = 2;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this.table = new Node[INITIAL_CAPACITY];
```

Comment:
Remember to use the diamond operator correctly. It should be 'new Node<>' to avoid raw type usage and ensure type safety.